charts/incubator/jdownloader2/SCALE/questions.yaml

```diff
@@ -0,0 +1,879 @@
+                        attrs:
+                          - variable: enabled
+                            label: "Enable the port"
+                            schema:
+                              type: boolean
+                              default: true
+                              hidden: true
+                          - variable: protocol
+                            label: "Port Type"
+                            schema:
+                              type: string
+                              default: "HTTP"
+                              enum:
+                                - value: HTTP
+                                  description: "HTTP"
+                                - value: "HTTPS"
+                                  description: "HTTPS"
+                                - value: TCP
+                                  description: "TCP"
+                                - value: "UDP"
+                                  description: "UDP"
+                          - variable: port
+                            label: "Container Port"
+                            schema:
+                              type: int
+                              default: 5800
+                              editable: false
+                              hidden: true
+                          - variable: targetport
+                            label: "Target Port"
+                            description: "This port exposes the container port on the service"
+                            schema:
+                              type: int
+                              default: 5580
+                              editable: true
+                              required: true
+                          - variable: nodePort
+                            label: "Node Port (Optional)"
+                            description: "This port gets exposed to the node. Only considered when service type is NodePort"
+                            schema:
+                              type: int
+                              min: 9000
+                              max: 65535
+                              default: 36093
+        - variable: myjd
+          label: "MyJDownloader"
+          description: "Port used by MyJDownloader mobile applications and browser extensions to establish a direct connect to the JDownloader"
+          schema:
+            type: dict
+            attrs:
+              - variable: enabled
+                label: "Enable the service"
+                schema:
+                  type: boolean
+                  default: true
+                  hidden: true
+              - variable: type
+                label: "Service Type"
+                description: "ClusterIP's are only internally available, nodePorts expose the container to the host node System, Loadbalancer exposes the service using the system loadbalancer"
+                schema:
+                  type: string
+                  default: "LoadBalancer"
```

Comment:
Why LoadBalancer by default @Ornias1993

Reply:
Because it allows access to lower ports than Nodeport.